CMakeLists.txt

```diff
@@ -14,6 +14,10 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/")
 
 option(BUILD_BOTH_STATIC_SHARED_LIB OFF)
 
+if (MSVC)
+	add_definitions(/D_CRT_SECURE_NO_WARNINGS=1)
```

Comment:
Wouldn't it be better to fix `sprintf` usage so as to catch cases where using sprintf would be dangerous?